models/SequenceVerificationDataCentric/models/SequenceVerificationDataCentric/SVDCdomain/AESequenceDC/SVDCclasses/ExtraJobInvariantDefn/ExtraJobInvariantDefn.masl

```diff
@@ -47,28 +47,20 @@ begin
+          Logger::log(Logger::Warning, "pv.sv.extrajobinvariantdefn.createsourceextrajobinvariantdefn", logMessage);
         end if;  
       else
-          logMessage := "AESequenceDC::ExtraJobInvariantDefn.createSourceExtraJobInvariantDefn : Invalid Event Definition provided (type 2) : " & invariantEventDefn.eventTypeName & "for Extra Job Invariant Name = " & invariantName;
-	      Logger::log(Logger::Error, "AESequenceDC", logMessage);
-	      eventContent := "FailureReason = Invalid Event Definition provided for Extra Job Invariant Name" & " : EventType = " & invariantEventDefn.eventTypeName & " : InvariantName = " & invariantName;
-	      Reporting~>reportEvent(Logger::Error, "svdc_job_definition_failed", eventContent);              
+        logMessage := "svdc_job_definition_failed : FailureReason = Invalid Event Definition provided for Extra Job Invariant Name" & " : EventType = " & invariantEventDefn.eventTypeName & " : InvariantName = " & invariantName;
+        Logger::log(Logger::Warning, "pv.sv.extrajobinvariantdefn.createsourceextrajobinvariantdefn", logMessage);
       end if;
     else
-      logMessage := "AESequenceDC::ExtraJobInvariantDefn.createSourceExtraJobInvariantDefn : Invalid Event Definition provided (type 1) : " & invariantEventDefn.eventTypeName & "for Extra Job Invariant Name = " & invariantName;
-	  Logger::log(Logger::Error, "AESequenceDC", logMessage);
-	  eventContent := "FailureReason = Invalid Event Definition provided for Extra Job Invariant Name" & " : EventType = " & invariantEventDefn.eventTypeName & " : InvariantName = " & invariantName;
-	  Reporting~>reportEvent(Logger::Error, "svdc_job_definition_failed", eventContent);
-	end if;              
+      logMessage := "svdc_job_definition_failed : FailureReason = Invalid Event Definition provided for Extra Job Invariant Name" & " : EventType = " & invariantEventDefn.eventTypeName & " : InvariantName = " & invariantName;
+      Logger::log(Logger::Warning, "pv.sv.extrajobinvariantdefn.createsourceextrajobinvariantdefn", logMessage);
+    end if;              
   else
-    logMessage := "AESequenceDC::ExtraJobInvariantDefn.createSourceExtraJobInvariantDefn : Invalid Job Definition provided : " & jobDefnName & "for Extra Job Invariant Name = " & invariantName;
-	Logger::log(Logger::Error, "AESequenceDC", logMessage);
-	eventContent := "FailureReason = Invalid Job Definition provided for Extra Job Invariant Name" & " : JobName = " & jobDefnName & " : InvariantName = " & invariantName;
-	Reporting~>reportEvent(Logger::Error, "svdc_job_definition_failed", eventContent);
+    eventContent := "svdc_job_definition_failed : FailureReason = Invalid Job Definition provided for Extra Job Invariant Name" & " : JobName = " & jobDefnName & " : InvariantName = " & invariantName;
```

Comment:
Should this be "logMessage := " instead of "eventContent := "?

Reply:
good catch!